Jenkinsfile

```diff
@@ -2,7 +2,7 @@
 
 
 def initializeEnvironment() {
-  def nodeVersions = ['8': '8.16.2', '10': '10.17.0', '12': '12.13.0']
+  def nodeVersions = ['8': '8.16.2', '10': '10.17.0', '12': '12.13.0', '16': '16.20.2', '18': '18.17.1', '20': '20.5.1']
```

Comment:
Along the same lines as the conversation in the runner image PR we should do away with anything < nodejs 16.x here.  So 8, 10 and 12 would go away outright.

Reply:
addressed in the latest commit